### ISSUE TITLE
Cifar Validation set

### ISSUE THREAD

**Issue by sxontheway** [KaihuaTang/Long-Tailed-Recognition.pytorch#31]
Hello. I found in `ImbalanceCIFAR.py`, the validation set and the test set seem to be the same. Is it better to use a balanced training set (50000 frames) as the validation set instead of using the 10000 frame test set for validation?  Thanks!

**Comment by KaihuaTang** [comment]
Thank you for your advice. We noticed the same problem when we were working on Imbalanced CIFAR. However, since this split is adopted by all the previous methods, we have to use the same split for fair comparisons.
